### ISSUE TITLE
[Bug] Error: Unable to access jarfile Tachidesk.jar

### ISSUE THREAD

**Issue by achempak** [AriaMoradi/Tachidesk#374]
## Device information
- Tachidesk version: Tachidesk-Server-v0.6.3-r1100-macOS-x64
- Server Operating System: MacOS Version 12.3.1
- Server Desktop Environment: N/A
- Server JVM version: bundled with release
- Client Operating System: MacOS Version 12.3.1
- Client Web Browser: 102.0.1 

## Steps to reproduce
1. Try to run any of the launcher scripts

### Expected behavior
The program should have launched

### Actual behavior
Received an error `Error: Unable to access jarfile Tachidesk.jar`

## Other details
I was able to resolve this by renaming the bundled jar file `Tachidesk-Server.jar` to `Tachidesk.jar`

**Comment by AriaMoradi** [comment]
Be kiram

**Comment by xeno-crimson** [comment]
I had a similar issue with `Tachidesk-Server-v0.6.3-r1100-linux-x64.tar.gz` I fixed it with by editing the script.

from this:
```sh
#!/bin/sh

exec /usr/bin/java -Dsuwayomi.tachidesk.config.server.webUIInterface=electron -Dsuwayomi.tachidesk.config.server.electronPath=/usr/bin/electron -jar /usr/share/java/tachidesk-server/tachidesk-server.jar
```

to this:
```sh
#!/bin/sh

exec ./jre/bin/java -Dsuwayomi.tachidesk.config.server.webUIInterface=electron -Dsuwayomi.tachidesk.config.server.electronPath=./electron/electron -jar ./Tachidesk-Server.jar
```

**Comment by xeno-crimson** [comment]
other scripts also had similar issue.
script is looking for stuff on /usr instead of the current directory.

**Comment by mahor1221** [comment]
> ### Expected behavior
> 
> The program should have launched
> ### Actual behavior
> 
> Received an error `Error: Unable to access jarfile Tachidesk.jar`
> ## Other details
> 
> I was able to resolve this by renaming the bundled jar file `Tachidesk-Server.jar` to `Tachidesk.jar`

I've introduced this bug on 291a23949acb4b3d114f45e7720130415476594b then fixed it on 7195a30d55daa9bc49d7ef2c4e6518568115c2bc and it will take effect on the next release. 

Thanks for reporting the issue 👍🏽

**Comment by mahor1221** [comment]
> other scripts also had similar issue. script is looking for stuff on /usr instead of the current directory.

That makes sense. I'll make a PR to solve it.
